cmd/critest/main.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"sort"
+
+	"github.com/golang/glog"
+	_ "github.com/kubernetes-incubator/cri-tools/pkg/validate"
```

Comment:
@feiskyer Add this line, so we will get `pkg/validate`, when we `go get github.com/kubernetes-incubator/cri-tools/cmd/cirtest`. And we will also get `ginkgo`, since `pkg/validate` depends on `ginkgo`.